mail_tracking/controllers/discuss.py

```diff
@@ -0,0 +1,28 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import http
+
+from odoo.addons.mail.controllers.discuss import DiscussController
+
+
+class MailTrackingDiscussController(DiscussController):
+    @http.route()
+    def mail_init_messaging(self):
```

Comment:
This method has a different signature (https://github.com/odoo/odoo/blob/913c8678cba0741e99610b1174ba6c42ef0c21a6/addons/mail/controllers/discuss.py#L194)
```suggestion
    def mail_init_messaging(self, **kwargs):
```